Code/GraphMol/MolOps.h

```diff
@@ -212,25 +234,32 @@ RDKIT_GRAPHMOL_EXPORT unsigned int getMolFragsWithQuery(
        - the caller is responsible for <tt>delete</tt>ing the pointer this
    returns.
  */
-RDKIT_GRAPHMOL_EXPORT ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
-                                   bool addCoords = false,
-                                   const UINT_VECT *onlyOnAtoms = nullptr,
-                                   bool addResidueInfo = false);
+inline ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
+                    bool addCoords = false,
+                    const UINT_VECT *onlyOnAtoms = nullptr,
+                    bool addResidueInfo = false) {
+  AddHsParameters ps{explicitOnly, addCoords, addResidueInfo};
+  RWMol *res = new RWMol(mol);
+  addHs(*res, ps, onlyOnAtoms);
+  return static_cast<ROMol *>(res);
```

Comment:
Can we use a `std::unique_ptr<RWMol>` and `release()`, just in case of errors?
(I'm already seeing myself making this change in the future, after an asan run...)

```suggestion
  std::unique_ptr<RWMol> res{new RWMol(mol)};
  addHs(res.get(), ps, onlyOnAtoms);
  return static_cast<ROMol *>(res.release());
```